### PR TITLE
Fix corresponding to issue #198

### PR DESCRIPTION
**Pull request by len0rd**
Added local rc_ovrd variable to do_roll_pitch_yaw_muxing to mirror do_throttle muxing. Also changed these local variable names for clarity, so they cant be confused with the static rc_override in the class. The mux loop in the mux_inputs function no uses the static rc_override data member.

### CODE REVIEW

File: src/mux.c [hunk 1]
```diff
@@ -175,36 +175,37 @@ static bool stick_deviated(mux_channel_t channel)
 
 static bool do_roll_pitch_yaw_muxing(mux_channel_t channel)
 {
+  bool rc_ovrd;
   //Check if the override switch exists and is triggered, or if the sticks have deviated enough to trigger an override
   if ((rc_switch_mapped(RC_SWITCH_ATT_OVERRIDE) && rc_switch(RC_SWITCH_ATT_OVERRIDE)) || stick_deviated(channel))
   {
-    rc_override = true;
```
Comment (superjax) [inline src/mux.c]: Why did you rename this variable?

Comment (len0rd) [inline src/mux.c]: Just for clarity, since the original issue was that function was writing to the static rc_override variable when it should have been writing to the local rc_override variable that it returns, as is done in the throttle muxing function. I just found it confusing having the local and static variables with the same name, so I decided to change it.

**Comment by superjax** [comment]
I think that this has been fixed in the latest rc1.0
